src/components/BalanceButton/index.tsx

```diff
@@ -9,26 +9,35 @@ export type Props = {
   disableHalfButton?: boolean
   disableMaxButton?: boolean
   onClickBalanceButton?: (value: BalanceButtonValueEnum) => void
+  height?: number
 }
 
-export default function BalanceButton({ disableHalfButton, disableMaxButton, onClickBalanceButton }: Props) {
+export default function BalanceButton({
+  disableHalfButton,
+  disableMaxButton,
+  onClickBalanceButton,
+  height,
+  ...rest
+}: Props) {
   const { t } = useTranslation()
 
   if (onClickBalanceButton == null) {
     return null
   }
 
   return (
-    <RowFlat style={{ marginLeft: '4px' }}>
+    <RowFlat style={{ marginLeft: '4px'}} {...rest}>
       <StyledBalanceLeftButton
         disabled={disableHalfButton}
         onClick={() => onClickBalanceButton(BalanceButtonValueEnum.HALF)}
+        height={height ?? 16}
```

Comment:
Wanna just make `height` an optional arg in `StyledBalanceButton`?

Reply:
addressed, replaced inline styles and height prop by styled components